### PR TITLE
Add how to get development provisioning profile

### PR DESCRIPTION
**Pull request by sraikimaxime**

### CODE REVIEW

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/sraikimaxime-patch-1/react-native/debugging/debug-javascript-ios-device.mo.md">debug-javascript-ios-device.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Review by tychota — APPROVED** [review]

**Comment by tychota** [comment]
@sraikimaxime @yannicktian @Almouro someone to be owner ?

**Comment by Almouro** [comment]
Why does this have Need fixing label? @yleflour @sraikimaxime

**Comment by yleflour** [comment]
@Almouro I think this addition should be canceled. I have differentiated the 2 steps setup and deploy in an upcoming version of the generator.

File: react-native/debugging/debug-javascript-ios-device.mo.md [hunk 1]
```diff
@@ -1,11 +1,30 @@
...
+  - Copy it in your ios/yourAppName/info.plist after the key `<key>CFBundleIdentifier</key>`
+  - Create a new lane in your FastFile :
+  
+```
+lane :get_dev_provisioning_profile do
+  match(type: 'development', shallow_clone: true)
+end
+```
+
+  - Execute your lane : `bundle exec fastlane get_dev_provisioning_profile`
```
Comment (DariusAf) [inline react-native/debugging/debug-javascript-ios-device.mo.md]: Ça bloquait, la solution était `bundle exec fastlane get_dev_provisioning_profile --env=staging`

**Pull request closed without merging**